### PR TITLE
fix: modal overflow

### PR DESCRIPTION
**Pull request by iago1501**
#### What problem is this solving?

Overflow when opening modal in admin/cms/pages, not giving a good usability

#### How should this be manually tested?

Click in add favicon and check the modal

[Beforer](https://master--storecomponents.myvtex.com/admin/cms/store)

image

[After](https://customheader--storecomponents.myvtex.com/admin/cms/store)

image

#### Type of changes

| ✔️  | Type of Change                                                                            |
| --- | ----------------------------------------------------------------------------------------- |
| ✔️  | Bug fix                                |
| \_  | New feature                        |
| \_  | Breaking change  |
| ✔️  | Technical improvements  |

### CODE REVIEW

**Review by beatrizmaselli — APPROVED** [review]
LBTM (looks better to me 😆 )

**Review by vsseixaso — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
+
+- Modal in admin/cms/store pages
```
Comment (vmourac-vtex) [inline CHANGELOG.md:12]:
```suggestion
- Favicon modal in admin/cms/store pages
```

**Review by vmourac-vtex — APPROVED** [review]
LGTM!  
Left an optional suggestion in the `CHANGELOG.md` for better clarity on what modal has been fixed

File: CHANGELOG.md (hunk 1 above)
Comment (iago1501) [reply CHANGELOG.md:12]: This is for every component using modal in this case. 

The favicon is only in this example

**Pull request merged**